programs/agora-court/src/instructions/initialize_case.rs

```diff
@@ -52,6 +54,8 @@ pub fn create_case(ctx: Context<CreateCase>, _dispute_id: u64, evidence: String)
         bump,
     });
     dispute.submitted_cases += 1;
+    // this doesn't prevent one person submitting multiple cases,
+    // I propose pre-initializing all cases in `initialize_dispute.rs`
```

Comment:
This still isn't accurate right? This should be handled by the fact that a PDA cannot be initialized twice.